pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -33,6 +34,8 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/feature"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/names"
+	"k8s.io/kubernetes/pkg/volume"
```

Comment:
There is some effort to move scheduler dependencies to staging. Are we able to move the methods we need to the same place?

https://github.com/kubernetes/kubernetes/pull/103062

Reply:
I took a look at #103062

It seems there are few changes they made to the utils. However, none of them are being used here. The volume package we are using here is to access the struct volume.Spec. If we want to decouple it, we can create new functions in the volumehelper I guess. I think it would be good to either wait for that change get merged and make another patch to do the refactor or we can make our change first and do that all together. Otherwise it might be inconsistent and have huge merge conflicts. Wdyt? I can do both.

Reply:
After looking into it, I managed to remove all the dependency added in this commit. It should be good now.